apps/server/src/modules/registration-pin/service/registration-pin.service.ts

```diff
@@ -1,11 +1,18 @@
 import { Injectable } from '@nestjs/common';
+import { LegacyLogger } from '@src/core/logger';
 import { RegistrationPinRepo } from '../repo';
 
 @Injectable()
 export class RegistrationPinService {
-	constructor(private readonly registrationPinRepo: RegistrationPinRepo) {}
+	constructor(private readonly registrationPinRepo: RegistrationPinRepo, private readonly logger: LegacyLogger) {
+		this.logger.setContext(RegistrationPinService.name);
+	}
 
 	async deleteRegistrationPinByEmail(email: string): Promise<number> {
-		return this.registrationPinRepo.deleteRegistrationPinByEmail(email);
+		this.logger.log({ action: 'Deleting registrationPin for ', email });
+		const result = this.registrationPinRepo.deleteRegistrationPinByEmail(email);
+		this.logger.log({ action: 'Deleted registrationPin for ', email });
```

Comment:
Please log some more metadata (as for the _classes_ cases where I've left some example).

Reply:
OK I can only now add info about number of deleted registrationPins for email. More deteiled information about updated documents will be added in ticket https://ticketsystem.dbildungscloud.de/browse/BC-6114